Configuration/TCA/Overrides/fe_users.php

```diff
@@ -168,8 +168,10 @@
         ],
     ]
 ];
-$GLOBALS['TCA']['fe_users']['columns']['image']['config']
-    = \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::getFileFieldTCAConfig(
+$typo3Version = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class)->getMajorVersion();
+if ($typo3Version < 12) {
```

Comment:
For another time, let's put in the "if" what is the most probable, meaning the current version of TYPO3, and in the "else" part what is more legacy...